example/metro.config.js

```diff
@@ -1,45 +1,18 @@
-const { getDefaultConfig, mergeConfig } = require('@react-native/metro-config');
 const path = require('path');
-const escape = require('escape-string-regexp');
-const exclusionList = require('metro-config/src/defaults/exclusionList');
-const pak = require('../package.json');
+const { getDefaultConfig } = require('@react-native/metro-config');
+const { getConfig } = require('react-native-builder-bob/metro-config');
+const pkg = require('../package.json');
 
 const root = path.resolve(__dirname, '..');
-const modules = Object.keys({ ...pak.peerDependencies });
 
 /**
  * Metro configuration
  * https://facebook.github.io/metro/docs/configuration
  *
  * @type {import('metro-config').MetroConfig}
  */
-const config = {
```

Comment:
All this config stuff was for turbo modules or something?

Reply:
Yeah more templated stuff, this is a file we've never touched so I felt confident upgrading to the latest implementation.
I think build tools have been moving around over the releases since we initialized our repo, and so probably this stuff has been internalized into other rn packages.